### PR TITLE
CBO-992: Stylesheet refactoring, remove inline styles

### PR DESCRIPTION
**Pull request by naseberry**
#### What
Refactor CCCD stylesheets by removing inline css styles.

#### Ticket
[Stylesheet refactoring, inline styles](https://dsdmoj.atlassian.net/browse/CBO-992)

#### Why
The separation of concerns, programming fundamental, to separate css from the view.

The majority of inline css was based on jquery `show()` and `hide()` methods, the PR replaces those methods with `removeClass()` and `addClass()`

### CODE REVIEW

File: app/assets/javascripts/modules/Modules.OffenceSearchView.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ moj.Modules.OffenceSearchView = {
     this.$view.find('.fx-filters-display p').append(this.filterResults(options));
 
     this.$view.find('.fx-results-found p').empty();
-    this.$view.find('.fx-results-found p').append('Results found: ' + options.results.length + ' <a href="#noop" class="fx-clear-search">Clear search</a>');
+    this.$view.find('.fx-results-found p').append('Results found: ' + options.results.length + ' <a href="#noop" class="fx-clear-search hidden">Clear search</a>');
```
Comment (colinbruce) [inline app/assets/javascripts/modules/Modules.OffenceSearchView.js:154]: Is this supposed to be hidden from the moment it's added?

File: app/assets/javascripts/modules/external_users/claims/BlockHelpers.js [hunk 2]
```diff
@@ -605,17 +605,18 @@ moj.Helpers.Blocks = {
...
-      $dom.find('.fx-travel-mileage-car input').is(function() {
+      $dom.find('.fx-travel-mileage-car input').is(function () {
         $(this).prop('disabled', !config.carModel);
 
       });
 
       // Bike mileage visibility, radio checked & disabled values
       $dom.find('.fx-travel-mileage-bike').css('display', config.bike);
-      $dom.find('.fx-travel-mileage-bike input[type=radio]').is(function() {
+      console.log(config);
```
Comment (colinbruce) [inline app/assets/javascripts/modules/external_users/claims/BlockHelpers.js]: Should we be outputting to console?

File: app/views/shared/_determinations_form.html.haml [hunk 3]
```diff
@@ -63,7 +63,7 @@
               = b.check_box(checked: params[:state_reason].present? && params[:state_reason].include?(b.value), 'aria-labelledby': "checkbox-control-legend-1 #{b.text.to_css_class}")
               = b.label { b.text }
 
-          .form-group.panel.nested-fields.fieldset.spacer.js-reject-reason-text{ style: 'display: none; padding-top: 10px;', class: error_class?(@error_presenter, :rejected_reason_other) }
+          .form-group.panel.nested-fields.fieldset.spacer.js-reject-reason-text.hidden{ class: error_class?(@error_presenter, :rejected_reason_other) }
```
Comment (colinbruce) [inline app/views/shared/_determinations_form.html.haml:66]: Padding-top being removed...

File: spec/javascripts/modules-amountAssessed_spec.js [hunk 4]
```diff
@@ -83,40 +83,40 @@ describe("Modules.AmountAssessedBlock.js", function() {
...
         var block = moj.Modules.AmountAssessed.blocks[0];
         var x = false;
-        $.fn.slideUp = function(){
+        $.fn.slideUp = function () {
           x = true;
         }
         expect(x).toEqual(false)
         block.slider(false, $('<div />'))
-        expect(x).toEqual(true)
+        expect(x).toEqual(false)
```
Comment (colinbruce) [inline spec/javascripts/modules-amountAssessed_spec.js]: Changing boolean output seems odd...

**Review by colinbruce — COMMENTED** [review]
Generally looks good, but there are a couple of observations/questions... happy to discuss

File: app/assets/javascripts/modules/Modules.OffenceSearchView.js (hunk 1 above)
Comment (naseberry) [reply app/assets/javascripts/modules/Modules.OffenceSearchView.js:154]: Yes, the default functionality is to hide the 'Clear search' link until there are search results.

File: app/views/shared/_determinations_form.html.haml (hunk 3 above)
Comment (naseberry) [reply app/views/shared/_determinations_form.html.haml:66]: Yes, padding top was removed from the view, all styles now reside in the scss.

File: app/assets/javascripts/modules/external_users/claims/BlockHelpers.js (hunk 2 above)
Comment (naseberry) [reply app/assets/javascripts/modules/external_users/claims/BlockHelpers.js]: Well spotted, we should never output to console.

**Review by colinbruce — APPROVED** [review]
After changes and discussions... :+1:

**Pull request merged**